modules/gluster/manifests/logging.pp

```diff
@@ -0,0 +1,42 @@
+# == Define: gluster::logging
+#
+# Read log file and sends to centralised logger (e.g reads /var/log/glusterfs/glusterd.log and sends to gluster).
+#
+# === Parameters
+#
+# [file_source_options*]
+#   Options for the file source for example [ '/var/log/glusterfs/glusterd.log' ].
+#   This essentially reads from the log file and sends it to gluster.
+#
+define gluster::logging (
+	Array[String] $file_source_options
+) {
+	syslog_ng::rewrite { 'r_application_name':
+		params => {
+			'type'      => 'set',
+			'options'   => [
+				'glusterd',
+				{ 'value' => 'APPLICATION_NAME' }
+			],
+		},
+	} ->
+	syslog_ng::source { "s_file_${title}":
+		params => {
+			'type'    => 'file',
+			'options' => $file_source_options,
```

Comment:
I have given a VALID array of options below:

'options' => [
    '/var/log/glusterfs/glusterd.log',
    { 'flags' => 'no-parse' }
]

Reply:
Fixed!